src/tokenizer.rs

```diff
@@ -149,6 +151,7 @@ impl fmt::Display for Token {
             Token::Number(ref n, l) => write!(f, "{}{long}", n, long = if *l { "L" } else { "" }),
             Token::Char(ref c) => write!(f, "{}", c),
             Token::SingleQuotedString(ref s) => write!(f, "'{}'", s),
+            Token::DoubleQuotedString(ref s) => write!(f, "{:?}", s),
```

Comment:
Shouldn't the display also print out the double quotes? Something like:

```suggestion
            Token::DoubleQuotedString(ref s) => write!(f, r#""{:?}""#, s),
```